api/http/management.go

```diff
@@ -371,20 +389,49 @@ Loop:
 				return err
 			}
 			if mr.Header.Proto == ws.ProtoTypeShell {
-				if mr.Header.MsgType == shell.MessageTypeShellCommand {
+				switch mr.Header.MsgType {
+				case shell.MessageTypeShellCommand:
+					timeNowUTC := time.Now().UTC().UnixNano()
+					keystrokeDelay := timeNowUTC - lastKeystrokeAt
+					if keystrokeDelay >= keyStrokeDelayRecordingThresholdMs {
+						controlMsg := app.Control{
+							Type:   app.DelayMessage,
+							Offset: recordedBytes,
+							DelayMs: uint16(float64(keystrokeDelay) *
+								0.000001),
```

Comment:
This will overflow if the user waits for ~66 seconds. Maybe truncate the delay to 65535ms if it exceeds this limit.

Reply:
makes sense!